nn_sdwan/manifest.json

```
@@ -0,0 +1,41 @@
+{
+  "display_name": "Netnology SDWAN",
+  "maintainer": "info@netnology.io",
+  "manifest_version": "1.0.0",
+  "name": "nn_sdwan",
+  "metric_prefix": "nn_sdwan.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "SDWAN Controller Metric Exporter by Netnology",
+  "guid": "a7c79ba9-a4b7-4fa4-b4aa-3868458e2101",
+  "support": "extras",
```

```suggestion
  "support": "contrib",
```
`extras` is not valid here. The choices are 'core', 'contrib', 'partner'